hadoop-tools/hadoop-aliyun/src/main/java/org/apache/hadoop/fs/aliyun/oss/AliyunOSSFileSystemStore.java

```diff
@@ -147,6 +147,13 @@ public void initialize(URI uri, Configuration conf, String user,
       throw new IllegalArgumentException(msg);
     }
 
+    boolean redirectEnable = conf.getBoolean(REDIRECT_ENABLE_KEY,
+    REDIRECT_ENABLE_DEFAULT);
+    if (!redirectEnable) {
```

Comment:
Could we just use  `clientConf.setRedirectEnable(redirectEnable);` ?
Current implementation always assumes `true` is default for clientConf, right ?

Reply:
1 I will optimize the code.
2 yes,the OSS SDK defaults the redirect setting to true